.eslintrc

```diff
@@ -0,0 +1,266 @@
+{
+    "parserOptions": {
+        "ecmaVersion": 6,
+        "sourceType": "module"
+    },
+    /*
+    * ENVIRONMENTS
+    * =================
+    */
+    "env": {
+
+        // Define globals exposed by modern browsers.
+        "browser": true,
+
+        // Allow CommonJS writing
+        "node": true,
+
+        // es2015
+        "es6":true,
+
+        // mocha test framework
+        "mocha":true
+    },
+
+    "globals": {
+
+    },
+
+    "rules": {
+
+        /*
+        * ENFORCING OPTIONS
+        * =================
+        */
+
+        // Prohibits use of == and != in favor of === and !==.
+        "eqeqeq": 2,
+
+        // Requires all for in loops to filter object's items.
+        "guard-for-in": 2,
+
+        // Prohibits overwriting prototypes of native objects such as Array, Date and so on.
+        "no-extend-native": 2,
+
+        // Prohibits use of a variable before it is defined.
+        "no-use-before-define": 2,
+
+        // Prohibits the use of arguments.caller and arguments.callee.
+        "no-caller": 2,
+
+        // Prohibits the use of the comma operator.
+        "no-sequences": 2,
+
+        // Warns about "non-breaking whitespace" characters.
+        "no-irregular-whitespace": 2,
+
+        // Prohibits the use of constructor functions for side-effects.
+        "no-new": 2,
+
+        // Nope, thanks to strict-loader for webpack & use-strict module for node
+        "strict": 0,
+
+        // Prohibits use of explicitly undeclared variables.
+        "no-undef": 2,
+
+        // Warns when variables are defined but never used.
+        "no-unused-vars": 2,
+
+        // requireCurlyBraces
+        // TODO Not completely compatible with original rule.
+        "curly": [
+            2,
+            "all"
```

Comment:
You can remote the TODO comment (in fact all the TODO comments) and replace `all` with `"multi-line", "consistent"`